app/views/inthenews/home.html.erb

```diff
@@ -1,15 +1,31 @@
 <%= render_overridable "partials", "inthenews_title" %>
 
-<div class="container-fluid home_container">
+</div>
+<div class="inthenews_background">
+  <div class="container lead">
+    <p>African Poetry and Poetics in the News  explores how depictions of African poetry in the news media have influenced and shaped perceptions of what African poetry is in the 21st century. It facilitates and supports current research relating to Poetry and Revolution, Race and Poetry, and Poetry and Migration as it relates to the African continent and cultures.</p>
+  </div>
+</div>
+<div class="container">
+
+<div class="container-fluid home_container inthenews_container">
```

Comment:
Can we look at indenting the content inside of these divs to keep the alignment below consistent?